homeassistant/components/efergy/sensor.py

```diff
@@ -60,12 +64,15 @@
         key=CONF_COST,
         name="Energy Cost",
         device_class=DEVICE_CLASS_MONETARY,
+        native_unit_of_measurement="USD",
```

Comment:
This won't be used since we set the unit attribute in the entity for the corresponding entity.